domainEntityFilters/www/filter.js

```diff
@@ -0,0 +1,74 @@
+//
+//  filter.js
+//
+//  Created by Zach Fox on 2019-06-07
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  See accompanying README.md for usage instructions.
+// 
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+
+function filterAdd(properties, originalProperties) {
+    if (properties.name.indexOf("Whiteboard") > -1) {
+        return properties;
+    }
+
+    return false;
+}
+
+
+function filterEdit(properties, originalProperties) {
+    if (originalProperties.name.indexOf("Whiteboard") > -1) {
+        return properties;
+    } else if (properties.privateUserData) {
+        var retProps = {
+            privateUserData: properties.privateUserData

```

Comment:
Have you had a chance to test if this works as expected?

Reply:
yup! and you helped! :D ty!